.github/dependabot.yml

```diff
@@ -8,6 +8,13 @@ updates:
       time: "09:00"
       timezone: "Europe/London"
 
+  - package-ecosystem: "npm"
+    directory: "/examples/express"
+    schedule:
+      interval: "daily"
+      time: "09:00"
+      timezone: "Europe/London"
```

Comment:
**question:** Am I right in thinking that the PR is not dependent on this change?

Reply:
Yes you are, oops! I'll extract this

Reply:
@rowanmanning But also should there be a dependabot change on this PR for the `app-info` package?

Reply:
Oh! Or not because it has no dependencies!

Reply:
Yep I feel like _maybe_ that'll trip us up if this package does ever have dependencies, but I think it's quite unlikely to ever need any?